class/defaults.yml

```diff
@@ -9,6 +9,9 @@ parameters:
       cert_manager:
         source: https://charts.jetstack.io
         version: v1.16.3
+      exoscale_webhook:
+        source: https://github.com/exoscale/cert-manager-webhook-exoscale.git
+        version: v0.3.0
```

Comment:
I'm pretty sure this will confuse commodore-renovate

Reply:
Fingers crossed 🤞